package.json

```diff
@@ -111,7 +111,7 @@
     "nock": "8.0.0",
     "npmshrink": "1.0.1",
     "nyc": "13.0.0",
-    "proxyquire": "1.8.0",
+    "proxyquire": "2.0.0",
```

Comment:
Purely for my own curiosity, are there any cool new features/fixes we're getting with this upgrade or is it just to get us on the latest and greatest generally?

Reply:
I believe this upgrade just had bug fixes. I thought they might resolve some of the flaky test but didn't.